src/components/DocumentProducer/EditImagesModal/PhotoSelector.tsx

```diff
@@ -150,6 +152,7 @@ export default function PhotoChooser(props: PhotoChooserProps): JSX.Element {
                   <PhotoPreview
                     imgUrl={fileData.url}
                     includeTrashIcon={true}
+                    imageWidth={previewWidth || 120}
```

Comment:
If previewWidth is initialized to 120 I don't think the `|| 120` is needed here

Reply:
Fixed!